conf/machine/imx7ulpevk.conf

```diff
@@ -9,7 +9,7 @@ MACHINEOVERRIDES =. "mx7:mx7ulp:"
 require conf/machine/include/imx-base.inc
 require conf/machine/include/tune-cortexa7.inc
 
-MACHINE_FEATURES += " pci wifi bluetooth qca9377"
+MACHINE_FEATURES += "pci wifi bluetooth bcm43430"
 
 KERNEL_DEVICETREE = " \
```

Comment:
Little confused here because we did in fact already have these all listed. I added `imx7ulp-com.dts` for `use-mainline-bsp`. Please check if I did this correctly, I didn't build mainline myself.